analyses/independent-samples-release/02-generate-independent-rnaseq.R

```diff
@@ -0,0 +1,83 @@
+# 02-generate-independent-rnaseq.R
+# Krutika Gaonkar for D3b
+#
+# Purpose: Generate tables of independent rna-seq specimens 
+# Option descriptions
+# -f, --histology_file : File path to where you would like the annotation_rds file to be
+#               stored
+# -o,--output_directory : Output directory
+# example invocation:
+# Rscript analyses/independent-samples/02-generate-independent-rnaseq.R \
+#   -f data/pbta-histologies.tsv \
+#   -o analyses/independent-samples/results
+
+# Load the libraries
+library(optparse)
+library(tidyverse)
+
+# Base directories
+root_dir <- rprojroot::find_root(rprojroot::has_dir(".git"))
+analysis_dir <- file.path(root_dir, "analyses", "independent-samples-release")
+
+# source sample selection function
+source(file.path(analysis_dir, "util", "independent-rna-samples.R"))
+
+set.seed(2020)
+
+# Parse options
+option_list <- list(
+  make_option(
+    c("-f", "--histology_file"),
+    type = "character",
+    default = NULL,
+    help = "path to the histology tsv file",
+  ),
+  make_option(
+    c("-o", "--output_directory"),
+    type = "character",
+    default = NULL,
+    help = "path to output directory"
+  ),
+  make_option(
+    c("-i","--independent_dna_sample_df"),
+    type = "character",
+    default = NULL,
+    help = "path to independent-specimens.wgs* file"
+  )
+)
+
+opts <- parse_args(OptionParser(option_list = option_list))
+
+# set output files
+out_dir <- opts$output_directory
+if (!dir.exists(out_dir)){
+  dir.create(out_dir, recursive = TRUE)
+}
+
+rnaseq_primplus_file <- file.path(out_dir, 
```

Comment:
change filename to include "release" to not overwrite the MTP indep specimen files